code/datums/elements/strippable.dm

```diff
@@ -368,6 +368,10 @@
 	if (.)
 		return
 
+	if (!iscarbon(usr) && !iscyborg(usr))
```

Comment:
note that this will prevent medborgs from doing some surgeries on dead/unconscious patients (like organ manipulation)